### PR TITLE
Add income 

### PR DESCRIPTION
**Pull request by EvgeniaSila**
- Created add income box and save income button 
- added total sum display 
- created methods to be later completed for calulcating total expenses

### CODE REVIEW

**Review by hellokathylee — DISMISSED** [review]

File: Uni_life_tracker/app/src/main/java/com/generic/plannr/ExpensesActivity.java [hunk 1]
```diff
@@ -76,10 +83,39 @@ public UserInfoDatabaseHelper createDatabase() {
         return user;
     }
 
+
```
Comment (dalshekerchi) [inline Uni_life_tracker/app/src/main/java/com/generic/plannr/ExpensesActivity.java]: Can you add javadocs here

File: Uni_life_tracker/app/src/main/java/com/generic/plannr/ExpensesActivity.java [hunk 2]
```diff
@@ -76,10 +83,39 @@ public UserInfoDatabaseHelper createDatabase() {
...
+    }
+
+
     public void clickAddExpense(View view) {
         // clicking the check in order to add expense
         Intent intent = new Intent(this, AddExpensesActivity.class);
         startActivity(intent);
+
+    }
+
```
Comment (dalshekerchi) [inline Uni_life_tracker/app/src/main/java/com/generic/plannr/ExpensesActivity.java]: Here as well

**Review by dalshekerchi — CHANGES_REQUESTED** [review]

File: Uni_life_tracker/app/src/main/java/com/generic/plannr/ExpensesActivity.java (hunk 2 above)
Comment (EvgeniaSila) [reply Uni_life_tracker/app/src/main/java/com/generic/plannr/ExpensesActivity.java]: got it

File: Uni_life_tracker/app/src/main/java/com/generic/plannr/ExpensesActivity.java (hunk 1 above)
Comment (EvgeniaSila) [reply Uni_life_tracker/app/src/main/java/com/generic/plannr/ExpensesActivity.java]: fixed

**Pull request merged**